### DISCUSSION TITLE
Consider making the `--all` option for `buildah manifest push` true by default

### DISCUSSION THREAD

**Discussion by k9withabone** [containers/buildah#5547] [Ideas]
While troubleshooting an issue with latest version of the Podlet container (containers/podlet#82), I found out that the `--all` option for `buildah manifest push` is not true by default like it is for `podman manifest push`. Please consider matching the behavior of `podman manifest push` for `buildah manifest push` to make switching between Podman and Buildah easier.

**Comment by rhatdan** [comment]
Please open a PR to change the behavior.

**Comment by k9withabone** [comment]
Closed with #5755.
